src/Workspaces/CSharp/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -244,6 +244,13 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 return true;
             }
 
+            // Switch expression arm
+            // x => (y)
+            if (node.Parent is SwitchExpressionArmSyntax)
+            {
+                return true;
```

Comment:
can you validate that you're on the "expression" side of the SwitchExpression? i.e.:

```c#
             if (node.Parent is SwitchExpressionArmSyntax arm && arm.Expression == node)
```